package/pom.xml

```diff
@@ -248,41 +238,47 @@
               <goal>run</goal>
             </goals>
           </execution>
+          <execution>
+            <id>rename war</id>
+            <phase>package</phase>
+            <configuration>
+              <target>
+                <move file="${distro.openmrsCoreDir}${file.separator}openmrs-webapp.war"
+                      tofile="${distro.openmrsCoreDir}${file.separator}openmrs.war"/>
+              </target>
+            </configuration>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
         </executions>
       </plugin>
 
       <!-- Assemble all Micro Frontends -->
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>exec-maven-plugin</artifactId>
-        <version>1.3.2</version>
         <executions>
           <execution>
-            <id>npm install openmrs</id>
+            <id>npx openmrs build</id>
             <goals>
               <goal>exec</goal>
             </goals>
-            <phase>initialize</phase>
+            <phase>compile</phase>
             <configuration>
-              <executable>npm</executable>
+              <executable>npx</executable>
               <arguments>
-                <argument>install</argument>
                 <argument>openmrs@${appShellVersion}</argument>
+                <argument>build</argument>
+                <argument>--spa-path</argument>
```

Comment:
I guess that will solve [the issue Burke and others](https://talk.openmrs.org/t/creating-a-build-pipeline-for-openmrs-3-0/33719/12?u=mksrom) had with environment vars being substituted at build time.